sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1007,6 +1007,7 @@ private void SetNameFieldsIfNull()
                 rangeGetContentHash.ToValidationOptions(),
                 progressHandler: default,
                 $"{nameof(BlobBaseClient)}.{nameof(Download)}",
+                false,
```

Comment:
I do think we should make all this null instead of false, so we don't have to re-record a bunch of tests